packages/client/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM node:12 AS builder
```

Comment:
How about using an `alpine` image here just like the api?
That doesn't work?

Reply:
alpine image doesn't seem to have all the dependencies to it needed to build the project. I found doing this is faster than installing those dependencies yourself.

Reply:
ok 👍

Reply:
Ninja (the build-system for BuckleScript) needs a couple of more libraries!